test/test_api/test_api.py

```diff
@@ -9,6 +9,13 @@
 from pytest import raises
 
 
+def _clear_last_exception_fields_in_sys():
```

Comment:
I'm not 100% sure what goes wrong. However running the 2.6 tests without this fails. As far as I've been able to see, it's because the last not-caught exception (which as far as I've been able to determine) is in `sys.last_*` fields. Since the 2.6 test skips the pep0484 tests, this then is filled with the skip-message, and importing the `sys` module results in some reference to something called `builtins.Skipped`, at which point it complains about not being able to find the builtins module.

This fix makes the test run, but possibly it masks one of 2 larger errors:
- I skip the python > 3.2 test in the wrong way (I put a string in IntegrationTestCase.skip)
- There is something else that goes wrong in importing the sys module in certain cases

To reproduce, run all tests `py.test test/` in python 2.6 (possibly any python < 3.2) without the changes in this file. Anything I can do to help further here, let me know!